Sample_Debugging_App/Properties/launchSettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "profiles": {
+    "Sample_Debugging_App": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{DD0A1BB6-11CE-11DD-8EE8-3F9E55D89593}",
+        "CORECLR_PROFILER_PATH_64": "C:\\Dev\\teamscale-profiler-dotnet\\Profiler\\bin\\Debug\\Profiler64.dll"
```

Comment:
Could we set the path by default to `..\..\..\..\Profiler\bin\Debug\Profiler64.dll` or too unstable? :grin: (For me the executable was under `Sample_Debugging_App\bin\Debug\net6.0`, so had to go up 4 folders :grin:)

Reply:
Ah that is great! Our documentation demands an "absolute path" at this point, so I wasn't sure if a relative path could work. I tried it and it works fine! :)